chain/jsonrpc/src/lib.rs

```diff
@@ -279,7 +279,15 @@ impl JsonRpcHandler {
             "query" => self.query(request.params).await,
             "status" => self.status().await,
             "tx" => self.tx_status_common(request.params, false).await,
-            "validators" => self.validators(request.params).await,
+            "validators" => {
```

Comment:
@frol should I introduce an alias here or is it fine as is?

Reply:
I think it is fine to modify the original method as long as we keep certain backward compatibility.